## no-hardlinks.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: we don't want and need hardlinks for those files,
## DP: especially because dh_strip chokes on them.

@DPATCH@
--- a/src/hd/Rmakefile
+++ b/src/hd/Rmakefile
@@ -14,6 +14,8 @@ MLIB = -lm
 
 CC = cc
 
+LN = ln
+
 SPECIAL =
 
 INSTDIR = /usr/local/bin
@@ -72,7 +74,7 @@ genrhgrid:	genrhgrid.o holo.o
 
 $(DEVDIR)/x11h.hdi:	$(DEVDIR)/x11.hdi
 	rm -f $(DEVDIR)/x11h.hdi
-	ln $(DEVDIR)/x11.hdi $(DEVDIR)/x11h.hdi
+	$(LN) x11.hdi $(DEVDIR)/x11h.hdi
 
 $(DEVDIR)/x11.hdi:	rhdisp.o rhdisp2.o viewbeams.o rhd_x11.o rhd_qtree.o \
 rhd_qtree2r.o rhd_ctab.o holo.o
@@ -81,35 +83,35 @@ rhd_x11.o rhd_qtree.o rhd_qtree2r.o rhd_
 
 $(DEVDIR)/glx1h.hdi:	$(DEVDIR)/glx1.hdi
 	rm -f $(DEVDIR)/glx1h.hdi
-	ln $(DEVDIR)/glx1.hdi $(DEVDIR)/glx1h.hdi
+	$(LN) glx1.hdi $(DEVDIR)/glx1h.hdi
 
 $(DEVDIR)/glxoh.hdi:	$(DEVDIR)/glxo.hdi
 	rm -f $(DEVDIR)/glxoh.hdi
-	ln $(DEVDIR)/glxo.hdi $(DEVDIR)/glxoh.hdi
+	$(LN) glxo.hdi $(DEVDIR)/glxoh.hdi
 
 $(DEVDIR)/glxsh.hdi:	$(DEVDIR)/glxs.hdi
 	rm -f $(DEVDIR)/glxsh.hdi
-	ln $(DEVDIR)/glxs.hdi $(DEVDIR)/glxsh.hdi
+	$(LN) glxs.hdi $(DEVDIR)/glxsh.hdi
 
 $(DEVDIR)/glxsoh.hdi:	$(DEVDIR)/glxso.hdi
 	rm -f $(DEVDIR)/glxsoh.hdi
-	ln $(DEVDIR)/glxso.hdi $(DEVDIR)/glxsoh.hdi
+	$(LN) glxso.hdi $(DEVDIR)/glxsoh.hdi
 
 $(DEVDIR)/oglh.hdi:	$(DEVDIR)/ogl.hdi
 	rm -f $(DEVDIR)/oglh.hdi
-	ln $(DEVDIR)/ogl.hdi $(DEVDIR)/oglh.hdi
+	$(LN) ogl.hdi $(DEVDIR)/oglh.hdi
 
 $(DEVDIR)/ogloh.hdi:	$(DEVDIR)/oglo.hdi
 	rm -f $(DEVDIR)/ogloh.hdi
-	ln $(DEVDIR)/oglo.hdi $(DEVDIR)/ogloh.hdi
+	$(LN) oglo.hdi $(DEVDIR)/ogloh.hdi
 
 $(DEVDIR)/oglsh.hdi:	$(DEVDIR)/ogls.hdi
 	rm -f $(DEVDIR)/oglsh.hdi
-	ln $(DEVDIR)/ogls.hdi $(DEVDIR)/oglsh.hdi
+	$(LN) ogls.hdi $(DEVDIR)/oglsh.hdi
 
 $(DEVDIR)/oglsoh.hdi:	$(DEVDIR)/oglso.hdi
 	rm -f $(DEVDIR)/oglsoh.hdi
-	ln $(DEVDIR)/oglso.hdi $(DEVDIR)/oglsoh.hdi
+	$(LN) oglso.hdi $(DEVDIR)/oglsoh.hdi
 
 $(DEVDIR)/glx1.hdi:	rhd_glx1.o rhdisp.o rhdisp2.o viewbeams.o \
 rhd_qtree.o rhd_qtree2c.o holo.o
